### DISCUSSION TITLE
Chain the states like hover, focus, etc

### DISCUSSION THREAD

**Discussion by TefDesign** [tailwindlabs/tailwindcss#7823] [Help]
Hello, 
I would like to know if it was possible to chain the states like `hover:focus:text-red-500` instead of repeating by writing : 
`hover: text-red-500 focus:text-red-500` ?

Thanks :)

**Comment by AviAvinav** [comment]
I don't think tailwind css has that feature, though I am not sure for v3, but you can use this chaining of states with [windicss](https://windicss.org/). It is built on top of tailwindcss, so you can migrate to it easily. Hope it helps.
